src/modules/core/components/popover/popover.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { useRef } from 'react';
+// components
+import { Typography } from '@/modules/core/components';
+
+import { Popover } from './popover';
+import { useBoolean } from 'usehooks-ts';
+
+const meta: Meta<typeof Popover> = {
+  title: 'Core UI/Popover',
+  component: Popover,
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof Popover>;
+
+export const Base: Story = {
+  render: () => {
+    const isPopoverOpen = useBoolean();
+    // refs
+    const buttonRef = useRef<HTMLButtonElement>(null);
+
+    return (
+      <>
+        <button ref={buttonRef} onClick={isPopoverOpen.setTrue}>
+          open
+        </button>
+        <Popover
```

Comment:
This popover has awful accessibility. The focusing inside power just won't work I suggest you to take a look at https://www.radix-ui.com/docs/primitives/components/popover.

Reply:
Thanks, I'll update my component. I prefer not to use any shit that becomes deprecated within 1-2 years.

Reply:
@IhorBalan Awful because it doesn't focus inside popover?

Reply:
The link was for you to see how it should work for accessibility. Better to update library once in 1-2 year than make complicate shit yourself and spend for this a lot of time, that can be put in different area.

Reply:
For the response to your second comment, yes, it's not focusing and if we will have some content there it will become totally unreachable by person who uses keyboard only.

Reply:
So, as for you this code is shit? Got you, thanks!

Reply:
It was a phrase, don't take it personally. What I want is for you to consider different approaches that already exist in the market and think if they will suit us better.

Reply:
Let's discuss it on the call.